### PR TITLE
feat(examples): new page for multiple configuration on paymentPlan Wid…

### PR DESCRIPTION
**Pull request by gautierlabarre**
- New page multiple widget example.
- Fix css for mobile testing
- clean up the html a little bit
- cdn line commented to toggle it faster during test

## Why another page ?

The idea is to make sure that the widgets work with multiple instance on the same page, and that the configuration can be different

### CODE REVIEW

**Review by benoitgrasset-alma — APPROVED** [review]

File: examples/multiple.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+          <div class="product">
+            <h1>Sac à main</h1>
+            <select onchange="generateRenderer()" id="quantity">
+              <option value="1">1</option>
+              <option value="2">2</option>
+              <option value="3">3</option>
+              <option value="10">10</option>
+            </select>
+            <h3>450 €</h3>
+            <p>Deluxe edition, bigger price, same product</p>
```
Comment (Taskim) [inline examples/multiple.html]: not sure to understand this ?

File: examples/multiple.html [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+            <div id="alma-widget-payment-plans"></div>
+            <h3>250 €</h3>
+            <p>Basic edition, lower price, smarter purchase</p>
```
Comment (Taskim) [inline examples/multiple.html]: and this

File: examples/multiple.html [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+          </div>
+          <div class="open-modal-ctas">
+            <a href="#" id="alma-open-modal-button">Open modal with clickableSelector</a>
+            <a href="#" onClick="openModal()">Open modal with open function</a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <script src="../dist/widgets.umd.js"></script>
+    <!-- <script src="https://cdn.jsdelivr.net/npm/@alma/widgets@2.5.1/dist/widgets.umd.js"></script> -->
```
Comment (Taskim) [inline examples/multiple.html]:
```suggestion
```

File: examples/style.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  .open-modal-ctas a:visited {
+    display: block;
+    margin: 8px 0;
+    font-size: 14px;
+    color: #00425d;
+  }
+  h1,
+  h3 {
+    margin: 16px 0;
+  }
```
Comment (Taskim) [inline examples/style.css]: missing CR

File: examples/basic.html [hunk 5]
```diff
@@ -6,89 +6,20 @@
...
-        margin: 8px 0;
-        font-size: 14px;
-        color: #00425d;
-      }
-      h1,
-      h3 {
-        margin-bottom: 16px;
-      }
-    </style>
+      href="https://cdn.jsdelivr.net/npm/@alma/widgets@2.5.1/dist/widgets.min.css"
```
Comment (Taskim) [inline examples/basic.html:12]: you should probably remove this ?

File: examples/basic.html [hunk 6]
```diff
@@ -116,9 +47,9 @@ <h3>450 €</h3>
           </div>
         </div>
       </div>
-      <div></div>
     </div>
     <script src="../dist/widgets.umd.js"></script>
+    <!-- <script src="https://cdn.jsdelivr.net/npm/@alma/widgets@2.5.1/dist/widgets.umd.js"></script> -->
```
Comment (Taskim) [inline examples/basic.html:52]: and this

File: examples/multiple.html [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Alma Widgets multiple configuration example</title>
+
+    <link rel="stylesheet" href="../dist/widgets.umd.css" />
+    <link rel="stylesheet" href="./style.css" />
+    <!-- <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/@alma/widgets@2.5.1/dist/widgets.min.css"
+    /> -->
```
Comment (Taskim) [inline examples/multiple.html]: and this :D

File: examples/multiple.html (hunk 1 above)
Comment (gautierlabarre) [reply examples/multiple.html]: I will change this p to have a text linked to the configuration of the widget then

File: examples/multiple.html (hunk 3 above)
Comment (gautierlabarre) [reply examples/multiple.html]: I actually want this line to be available as we need to often switch between the two. This allow a comment/uncomment really fast.

Also our clients use the cdn so to test a specific version it's easier.

File: examples/basic.html (hunk 5 above)
Comment (gautierlabarre) [reply examples/basic.html:12]: Same, useful to switch easily and test specific version

**Review by Taskim — APPROVED** [review]

File: examples/multiple.html (hunk 3 above)
Comment (gautierlabarre) [reply examples/multiple.html]: As discussed, I keep the comment

File: examples/basic.html (hunk 5 above)
Comment (gautierlabarre) [reply examples/basic.html:12]: As discussed, I keep the comment

File: examples/multiple.html (hunk 7 above)
Comment (gautierlabarre) [reply examples/multiple.html]: As discussed, I keep the comment

File: examples/basic.html (hunk 6 above)
Comment (gautierlabarre) [reply examples/basic.html:52]: As discussed, I keep the comment

**Pull request merged**